qemu/tests/cpuflags.py

```diff
@@ -404,7 +404,7 @@ def install_cpuflags_test_on_vm(vm, dst_dir):
         vm.copy_files_to(cpuflags_src, dst_dir)
         session.cmd("sync")
         session.cmd("cd %s; make EXTRA_FLAGS='';" %
-                    os.path.join(dst_dir, "cpu_flags"))
+                    os.path.join(dst_dir, "src"))
```

Comment:
why not use a variable for the path?

Reply:
this commit is a bugfix, not a style-fix. Anyway I don't really see an urgent need to turn this into a variable, usually this should not be changed. The reason it changed is that files got moved within sources.